app/src/main/res/layout/my_tasks.xml

```diff
@@ -49,17 +47,49 @@
         app:layout_constraintStart_toStartOf="parent" />
 
     <ImageView
-        android:id="@+id/imageView"
+        android:id="@+id/setingsButton"
```

Comment:
Double-check the spelling here

Reply:
yup that was what caused it lol. Thank you!